_posts/patterns/2015-12-21-lists.md

```diff
@@ -0,0 +1,274 @@
+</div>
+
+<div class="example-set">
+    <ul class="list-bulleted">
+        <li class="item">Apples</li>
+        <li class="item">Apricots</li>
+        <li class="item">Bananas</li>
+        <li class="item">Blackberries</li>
+        <li class="item">Blueberries</li>
+        <li class="item">Cherries</li>
+        <li class="item">Coconuts</li>
+        <li class="item">Cranberries</li>
+        <li class="item">Figs</li>
+        <li class="item">Oranges</li>
+        <li class="item">Plums</li>
+    </ul>
+
+    <ul class="list-bulleted list-circle">
+        <li class="item">Blueberries</li>
+        <li class="item">Cherries</li>
+        <li class="item">Coconuts</li>
+        <li class="item">Cranberries</li>
+    </ul>
+
+    <div class="list-bulleted" aria-role="list">
+        <p class="item" aria-role="listitem">Designed on top of an ocean, the town of Darnassea is home to dark elves lead by Colonel Alduin. This town wasn't built by an ocean by accident, as it has ancient, lost technologies, which is of great importance to the people of Darnassea and its success.</p>
+        <p class="item" aria-role="listitem">Darkwell has a troubled economy, which is mainly supported by jewelcrafting, fishing and fletching. But their biggest strengths are intricate jewelcrafting and rare herbalism. However, Darkwell lacks people skilled in farming.</p>
+        <p class="item">Raised inside a geyser field, the port of Darkwell is home to vikings lead by Mrs. Chilson. This port wasn't built by a geyser field by accident, as it has rare resources, which is of great importance to the people of Darkwell and its success.</p>
+    </div>
+
+    <ul class="list-bulleted list-disc no-indent">
+        <li class="item">Figs (No Indent)</li>
+        <li class="item">Oranges (No Indent)</li>
+        <li class="item">Plums (No Indent)</li>
+    </ul>
+</div>
+
+<h3 class="hd-6 example-set-hd">Ordered Lists</h3>
+
+<div class="example-set-description copy copy-meta">
+    <p>Ordered lists show a sequentially-based association between items. Its important to note that list-based styling can be applied to non-semantic list markup, but only when appropriate (and when leveraging any necessary aria-role attributes).</p>
+</div>
+
+<div class="example-set">
+    <ol class="list-ordered">
+        <li class="item">Apples</li>
+        <li class="item">Apricots</li>
+        <li class="item">Bananas</li>
+        <li class="item">Blackberries</li>
+        <li class="item">Blueberries</li>
+        <li class="item">Cherries</li>
+        <li class="item">Coconuts</li>
+        <li class="item">Cranberries</li>
+        <li class="item">Figs</li>
+        <li class="item">Oranges</li>
+        <li class="item">Plums</li>
+    </ol>
+
+    <div class="list-ordered" aria-role="list">
```

Comment:
I noticed there are zeroes before each item.  If you want to increment these, you can try something like this:
```
div.list-ordered {
    list-style-type: none;
    counter-reset: ol-counter;
}
div.list-ordered > p:before {
    content: counter(ol-counter);
    counter-increment: ol-counter;
}
```

I don't think the default spacing is any good, so you'll have to add sufficient padding to satisfy your visual needs.